levels/inside_perimeter/maps/mainhall.json

```diff
@@ -0,0 +1,753 @@
+{ "compressionlevel":-1,
```

Comment:
For this map, I see that we have `mainhall.png`, `mainhallv2.png`, and `mainhallv3.png`. Are we using all 3 of these?